.eslintrc

```diff
@@ -0,0 +1,29 @@
+{
+  "extends": "airbnb-base",
+  "env": {
+    "node": true,
+    "mocha": true
+  },
+  "rules": {
+    "arrow-body-style": ["warn", "as-needed"],
+    "comma-dangle": 0,
+    "global-require": "warn",
+    "import/no-unresolved": "off",
+    "new-cap": "off",
+    "no-param-reassign": 0,
+    "no-restricted-syntax": "off",
+    "no-return-assign": "warn",
+    "no-shadow": "off",
+    "no-underscore-dangle": "off",
+    "no-unused-expressions": "off",
+    "no-unused-vars": "warn",
+    "no-useless-escape": "off",
+    "no-trailing-spaces": ["error", { "skipBlankLines": true }],
+    "max-len": [ "warn", 120, 4 ],
+    "consistent-return": "warn",
+    "prefer-template": "warn",
+    "radix": "off",
+    "guard-for-in": "warn",
+    "spaced-comment":"off",
+  }
+}
```

Comment:
Any word on which `.eslint` ruleset we'd prefer to stick with going forward? I believe three exist in the wild at the moment